src/open_inwoner/openzaak/import_export.py

```diff
@@ -82,16 +81,25 @@ def from_exception_and_jsonl(cls, exception: Exception, jsonl: str) -> Self:
 
         return cls(error_template % error_data)
 
+    @classmethod
+    def from_jsonl(cls, jsonl: str) -> Self:
+        error_data = cls.extract_error_data(jsonl=jsonl)
+        error_template = (
+            "%(source_config_name)s was processed multiple times because it contains "
+            "duplicate natural keys: %(info)s"
```

Comment:
Perhaps something a bit more contextual:
```suggestion
            "%(source_config_name)s was processed multiple times because it shares "
            "defining attributes (such as `rsin`, `identificatie`, and `omschrijving`) with multiple objects. This may lead to unexpected results, as only the last row will have an effect on the target: %(info)s"
```

On the other hand, maybe your shorter and punchier version is better, as the most likely course is now that people fix it but escalate to us, and the main thing is that _we_ know what is going on. What do you think?

Reply:
Ignore more comment: I think your error message looks fine tbh.